apps/condo/domains/ticket/hooks/useTicketCancelModal.tsx

```diff
@@ -36,12 +31,12 @@ export const useTicketCancelModal: useTicketCancelModalType = (updateTicket) =>
 
     const cancelTicketModal = useMemo(() => (
         <Modal
-            visible={isModalVisible}
```

Comment:
You can use `Modal.useModal`, and then just spawn it dinamically. That's recommended option if source of opening = user interaction. (you do not need to destroy it manually)